src/components/book.js

```diff
@@ -1,38 +1,266 @@
 import React from 'react';
 import PropTypes from 'prop-types';
+import styled from 'styled-components';
+import { CircularProgressBar } from '@tomik23/react-circular-progress-bar';
 import { useDispatch } from 'react-redux';
 import { removeBookFromApi } from '../redux/books/thunkFunctions';
+import useWindowSize from './useWindowSize';
 
 function Book(props) {
   const {
-    id, title, author, catagory,
+    id, title, author,
   } = props;
   const dispatch = useDispatch();
   const handleRemove = () => {
     dispatch(removeBookFromApi(id));
   };
 
+  const isDesktop = useWindowSize();
+
+  const Progress = Math.floor(Math.random() * 100 + 1);
+  const currentChapter = Math.floor(Math.random() * 10);
+
   return (
-    <div>
-      <div>
-        <span>{catagory}</span>
-        <li key={id}>
-          {title}
-          {' '}
-          :
-          {author}
-        </li>
-        <button onClick={handleRemove} type="button"> Remove</button>
-      </div>
-    </div>
+    <BookList>
+      <ListItem key={id}>
+        <BookInfoContainer>
+          <Category>Fiction</Category>
+          <Title>{title}</Title>
+          <Author>{author}</Author>
+
+          <BookOptions>
+            <li>
+              <BookOption type="button">Comments</BookOption>
+            </li>
+            <li>
+              <BookOption type="button" onClick={handleRemove}>
+                Remove
+              </BookOption>
+            </li>
+            <li>
+              <BookOption className="edit" type="button">
+                Edit
+              </BookOption>
+            </li>
+          </BookOptions>
+        </BookInfoContainer>
+
+        <BookProgressContainer>
+          <CircularProgressBar
+            size={60}
+            animationSmooth="1s ease-out"
+            colorCircle="#e6e6e6"
+            linearGradient={['yellow', '#ff0000']}
+            round
+            percent={Progress}
+            number={!isDesktop}
+          />
+          {isDesktop && (
+            <ProgressNumContainer>
+              <ProgressStatistics>
+                {Progress}
+                %
+              </ProgressStatistics>
+              <ProgressCompleted>Completed</ProgressCompleted>
+            </ProgressNumContainer>
+          )}
+        </BookProgressContainer>
```

Comment:
- [ ] I would suggest you make your design similar to the Figma design as it is the requirement. Kindly make changes to the progress bar.
### your design
![your](https://user-images.githubusercontent.com/81727988/180430615-a64c006d-7703-482d-84ec-da41f834eb3e.png)
### Figma design
![figma](https://user-images.githubusercontent.com/81727988/180430671-500fd307-2732-44a1-9986-755d05989be0.png)